examples/amp-smartlinks.html

```diff
@@ -0,0 +1,49 @@
+<!--
+	Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+			http://www.apache.org/licenses/LICENSE-2.0
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS-IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the license.
+-->
+<!--
+	Test Description:
+	Valid amp-smartlinks tag
+-->
+<!doctype html>
+<html ⚡>
+<head>
+	<meta charset="utf-8">
+	<link rel="canonical" href="./regular-html-version.html">
+	<meta name="viewport" content="width=device-width,minimum-scale=1">
+	<meta name="amp-link-rewriter-priorities" content="amp-smartlinks">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
```

Comment:
add space(s) to format correctly here and below

Reply:
Thanks for the feedback. I'm not sure what you mean by add spaces. Comparing this boilerplate to other files in `examples/` it seems identical. Could you clarify what you mean by add spaces?

Reply:
...
    <meta name="amp-link-rewriter-priorities" content="amp-smartlinks">
    <style amp-boilerplate>..
    <script async custom-element="amp-smartlinks"...
    <script async src="https://cdn.ampproject.org/v0.js"></script>

style tag and the script referernce to v0 is not aligned with the rest of the markup.

Reply:
🤦‍♂️ completely misinterpreted your comment. Fixed!